roles/openafs_server/tasks/server-key.yaml

```diff
@@ -23,17 +21,16 @@
     dest: "{{ afs_afsconfdir }}/rxkad.keytab"
     remote_src: yes
   register: keytab
-  notify:
-    - Restart OpenAFS servers
-
+  
 - name: Check for akeyconvert availability
   stat:
-    path: "{{ afs_akeyconvert }}"
-  register: akeyconvert
+    path: "{{ afs_asetkey }}"
```

Comment:
name: is not correct now, and more importantly, we dont need to check for asetkey. we have to check for akeyconvert because it is "new" and is not available in older versions.

We should use akeyconvert for modern keys, and asetkey for legacy DES keys.